opendatahub-io/odh-manifests
# Update odh-dashboard manifests for dashboard v2.0 release

This updates the odh-dashboard manifests to deploy the odh-dashboard v2.0 release.  For the default dashboard deployment we will enable the `authentication` overlay in the dashboard by default.  

ALSO, this PR will set defaults for the jupyterhub-defaults-groups configmap so that `system:authenticated` users can login to JupyterHub. JH admins and ODH Dashboard Admin will use the `odh-jupyterhub-admins` as the default user group for admin access

Test steps:
1. Deploy odh-dashboard with `authentication` enabled
```yaml
  - kustomizeConfig:
      overlays:
        - authentication
      repoRef:
        name: manifests
        path: odh-dashboard
    name: odh-dashboard
```
2. Verify that dashboard deploys and `jupyterhub` is displayed as an enabled tile
